tests/integration/buildcmd/test_build_terraform_applications.py

```diff
@@ -249,16 +308,51 @@ def test_build_and_invoke_lambda_functions(self, function_identifier):
         self.assertEqual(return_code, 0)
 
         self._verify_invoke_built_function(
-            template_path=str(Path(self.terraform_application_path) / ".aws-sam/build/template.yaml"),
             function_logical_id=function_identifier,
             overrides=None,
-            expected_result={"statusCode": 200, "body": "[]"},
+            expected_result={
+                "statusCode": 200,
+                "body": "Hello, My friend!",
+                "headers": None,
+                "multiValueHeaders": None,
+            },
         )
 
 
-class TestBuildTerraformApplicationsWithImageBasedLambdaFunctionAndLocalBackend(BuildTerraformApplicationIntegBase):
-    pass
+@skipIf(
+    not CI_OVERRIDE,
+    "Skip Terraform test cases unless running in CI",
+)
+class TestBuildTerraformApplicationsWithImageBasedLambdaFunctionAndS3Backend(
+    BuildTerraformApplicationS3BackendIntegBase
+):
+    terraform_application = Path("terraform/image_based_lambda_functions_local_backend")
```

Comment:
`TestBuildTerraformApplicationsWithImageBasedLambdaFunctionAndS3Backend` and terraform application seems to contradict, because the terraform application still talks about a `local_backend` , am I missing something?

Reply:
my bad. I corrected the path, and tested it in my laptop.